src/components/HomePage.jsx

```diff
@@ -17,4 +18,10 @@ class HomePage extends Component {
   }
 }
 
+HomePage.propTypes = {
+  // Passed through ...Props
+  children: PropTypes.array,
```

Comment:
I have 

`propertyTemplates: PropTypes.arrayOf(PropTypes.object).isRequired` in mine -- dunno if that is right here, for the children thingy.

Reply:
After doing some testing/pairing with @jgreben yesterday, `PropTypes.object` may be too generic as about everything in Javascript is considered an object including strings and integers. Following @SaravShah suggestion of PropTypes.shape for many of the props, or look for specific Nodes or Elements.